packages/modular-scripts/src/cli.ts

```diff
@@ -304,7 +307,20 @@ function build(appPath: string) {
   });
 }
 
+type VerifyPackageTree = () => void;
+
+function check() {
+  const verifyPackageTree = require('react-scripts/scripts/utils/verifyPackageTree') as VerifyPackageTree; // eslint-disable-line @typescript-eslint/no-var-requires
+  verifyPackageTree();
+  // Unlike CRA, let's NOT verify typescript config when running tests.
+  console.log(chalk.greenBright(`All Good!`));
+}
+
 try {
+  if (process.env.SKIP_PREFLIGHT_CHECK !== 'true') {
+    check();
```

Comment:
Calling this here will mean that every run of modular-scripts will start with "All Good!", but I think we want that only when check is called manually (especially because it'll then confuse someone of the following command fails). 

Also, calling modular check itself will output "All good!" twice. 

Further, the other scripts (build, start) calls this already within react-scripts. So if there's a failure it'll appear twice. So I don't think it's a good idea to call this by default for every script.

Maybe we shouldn't implement modular check right now (or rather, maybe verifyPackageTree() isn't the right implementation for it), or rather shouldn't be in this PR. Let's instead just solve the problem of errors being swallowed. Just call verifyPackageTree() in this process for test()?

Reply:
> Also, calling modular check itself will output "All good!" twice.

lol I was so confused why this was happening - didn't think about it.

> Further, the other scripts (build, start) calls this already within react-scripts. So if there's a failure it'll appear twice. So I don't think it's a good idea to call this by default for every script. 

This is a good point - but verify will call `process.exit` if there is an issue, but I agree calling it twice is not ideal. 

> Just call verifyPackageTree() in this process for test()?

Yeah - I think for now this is the best implementation.